test/e2e/framework.go

```diff
@@ -1260,18 +1273,40 @@ func (data *TestData) gracefulExitAntreaAgent(covDir string, nodeName string) er
 		if err != nil {
 			return fmt.Errorf("error when graceful exit Antrea agent: copy pod files out, error:%v", err)
 		}
+
 	}
 	return nil
 }
 
+//collectAntctlCovFiles collects coverage files for antctl binary from pod and save to coverage directory
+func (data *TestData) collectAntctlCovFiles(podName string, containerName string, nsName string, covDir string) error {
+	// copy antctl coverage files out
+	cmds := []string{"bash", "-c", "find . -maxdepth 1 -name 'antctl*' -exec basename {} \\;"}
```

Comment:
I don't think `find` is a bash built-in command, so you should be able to use `find` as a the executable directly (maybe with one less backslash to escape the `;`)

Reply:
when run e2e on kind cluster, when finding the coverage .out files from master node, 
if use cmd "find . -maxdepth 1 -name 'antctl*.out' -exec basename {} ';'", the stdout is "find: missing argument to `-exec'".
but if  I changed to "bash -c find . -maxdepth 1 -name 'antctl*.out' -exec basename {} ';'", it can return the coverage .out file.

So used bash.